Makefile.am

```diff
@@ -204,6 +205,48 @@ ci:
 		exit $$status; \
 	fi
 
+check-branching:
+# checking if branching can be finished and all the pieces are in place
+	@echo "===================================="
+	@echo "Testing branch configuration"
+	@echo "===================================="
+	@if [ "$(SKIP_BRANCHING_CHECK)" != "false" ]; then \
+		echo "skipping..."; \
+	else \
+		CURRENT_BRANCH=$$(git rev-parse --abbrev-ref HEAD); \
+		if [ "$$CURRENT_BRANCH" != "$(GIT_BRANCH)" ]; then \
+			echo "*** Configuration is not adjusted for current branch!"; \
+			echo "This check will work only on main development branch fXX-release, rhel-X ..."; \
+			echo "current branch:  $$CURRENT_BRANCH"; \
+			echo "expected branch: $(GIT_BRANCH)"; \
+			exit 1; \
+		fi; \
+	fi
+
+	@echo "===================================="
+	@echo "Testing localization for $(L10N_DIR)"
+	@echo "===================================="
+	@curl --fail --silent --show-error -L https://raw.githubusercontent.com/rhinstaller/anaconda-l10n/$(GIT_L10N_BRANCH)/$(L10N_DIR)/anaconda.pot >/dev/null ||  \
+		(echo "*** Can't download translations from a localization repository"; \
+		 echo "repository: $(L10N_REPOSITORY)"; \
+		 echo "branch:     $(GIT_L10N_BRANCH)"; \
+		 echo "directory:  $(L10N_DIR)"; \
+	   exit 2)
```

Comment:
Indentation... ;-)

Reply:
Broken because I'm messing with tabs and spaces... Hmm I should be consistent.

Reply:
If that's the only problem with the code, you're doing something right!

Reply:
You know, it could be because people saw that bashisms everywhere and rather avoid looking on the code :D

Reply:
Fixed.